code/API_definitions/Application-endpoint-registration.yaml

```diff
@@ -1,22 +1,45 @@
 openapi: 3.0.3
 info:
-  title: Application Endpoint Registration API
-  version: 0.1.0
-  license:
-    name: Apache 2.0
-    url: https://www.apache.org/licenses/LICENSE-2.0.html
+  title: Application Endpoints Registration
   description: |
-    Application endpoint registration allows application developers to
-    register one or more Application Endpoints, and retrieve, update,and
-    delete those registrations.
+    The Application Endpoints Registration API provides a programmable
+    interface for developers to register the endpoints of an application
+    which is deployed across various Edge Cloud Zones.
+    It allows developers to retrieve, update, and delete those registrations.
+
+    # Introduction
+
+    The API registers the Edge Application Endpoint.
```

Comment:
Should we keep plural form in endpoint i.e. endpoints assuming applications may have one or more endpoints?

Reply:
Modified "endpoint" to "endpoints".